src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/utils/LoggingHelper.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.blobrouter.tasks.utils;
+
+import org.slf4j.Logger;
+
+public final class LoggingHelper {
+
+    private LoggingHelper() {
+        // util class
+    }
+
+    public static void wrapWithJobLog(Logger logger, String taskName, Runnable runnable) {
```

Comment:
I'm open to suggestions on alternative name :)

Reply:
`executeWithJobLog` ?

Reply:
I'd be even fine with `run`.